scripts/gcp-onboarding/README.md

```diff
@@ -72,15 +72,21 @@ Make sure you have below permission on GCP projects while executing the below co
 Execute the below command to promote the GCP service account at an organization level. It also assigns required ZCSPM roles to the service account
 
 ```bash
-$ ./promote-service-account.sh -s <SERVICE_ACCOUNT_EMAIL> -o <GCP_ORGANIZATION_ID>
+./promote-service-account.sh -s <SERVICE_ACCOUNT_EMAIL> -o <GCP_ORGANIZATION_ID>
 ```
 
 **Step 3: Enable Service APIs on GCP project**
 
+Execute the below command to enable ZCSPM required service APIs on single or multiple GCP projects
```

Comment:
Move this below organization command

[Optional] In case you want to enable ZCSPM required service APIs on single or multiple GCP projects present within the organization then execute the below command

Reply:
Resolved the comment